CHANGELOG.md

```diff
@@ -2,49 +2,66 @@
 
 All notable changes to this project will be documented in this file. Dates are displayed in UTC.
 
-#### [0.8.1](https://github.com/SatelCreative/spylib/compare/0.8.0...0.8.1)
+Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
```

Comment:
cool, generating the changelogs automatically.
Maybe update the [doc](https://github.com/SatelCreative/spylib/blob/4a146e35542923e333b631463f53c465f93f59bb/docs/development-contributing.md?plain=1#L68-L69) about it as well

Reply:
I thought we did already [here](https://satelcreative.github.io/spylib/development-contributing/#publishing-to-pipy)

Reply:
Now I am looking at the development-contribution doc [history](https://github.com/SatelCreative/spylib/commits/main/docs/development-contributing.md), the changelog was actually changed in PR #160  from using the auto-changelog to the github release auto release notes....  
you might be looking at an older doc that's not versioned. the latest is [0.8.1](https://satelcreative.github.io/spylib/0.8.1/development-contributing/)

Reply:
it's better to remove that [old doc](https://satelcreative.github.io/spylib/development-contributing/#publishing-to-pipy) somehow to prevent future confusion. is it a github page?

Reply:
Ok good catch! I will fix the link in CONTRIBUTING.md. That's what threw me off!

Reply:
Thank you for paying attention this! :pray: